kv/kvStore.go

```diff
@@ -0,0 +1,34 @@
+package kv
+
+import (
+	"errors"
+	"fmt"
+)
+
+const MaxMem = 1 << (10 * 3) // 1 GB
+const defPath = "."          // create in local directory
+
+type KeyValueStore interface {
+	Put(a1 uint64, a2 [10]byte)
+	Get(a1 uint64) [10]byte
+	Create(config KvStoreConfig)
+	Open(path string)
+	Delete(path string)
```

Comment:
Might have to return an error as file IO involved.
```suggestion
	Delete(path string) error
```